ydb/_topic_reader/topic_reader_asyncio_test.py

```diff
@@ -358,6 +358,32 @@ async def test_close_ack_waiters_when_close_stream_reader(
         with pytest.raises(topic_reader_asyncio.TopicReaderCommitToExpiredPartition):
             waiter.future.result()
 
+    async def test_flush(
+        self, stream, stream_reader_started: ReaderStream, partition_session
+    ):
+        offset = self.partition_session_committed_offset + 1
+        waiter = partition_session.add_waiter(offset)
+
+        stream.from_server.put_nowait(
+            StreamReadMessage.FromServer(
+                server_status=ServerStatus(ydb_status_codes_pb2.StatusIds.SUCCESS, []),
+                server_message=StreamReadMessage.CommitOffsetResponse(
+                    partitions_committed_offsets=[
+                        StreamReadMessage.CommitOffsetResponse.PartitionCommittedOffset(
+                            partition_session_id=partition_session.id,
+                            committed_offset=offset,
+                        ),
+                    ]
+                ),
+            )
+        )
+
+        await stream_reader_started.flush()
```

Comment:
It will good if wait small time to flush before send ack from a server and check for flush() not returned, then send ack and wait for flush completed.